md-dao/src/main/java/com/wipro/ats/bdre/md/dao/InstanceExecDAO.java

```diff
@@ -178,17 +178,24 @@ public void delete(Long id) {
                 List<InstanceExec> instanceExecList=instanceExecListCriteria.list();
                 long sumTime=0;
                 long currentTime=0;
+                boolean processRunning = false;
+                int stateOfProcess = 0;
+                int running = 0;
+                int failed = 0;
                 int total=instanceExecList.size();
                 for (InstanceExec instanceExec:instanceExecList)
                 {
                     if(instanceExec.getEndTs()!=null){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)